packages/@sanity/schema/src/sanity/extractSchema.ts

```diff
@@ -180,6 +180,11 @@ export function extractSchema(
       return {type: 'unknown'} satisfies UnknownTypeNode // we don't support cross-dataset references at the moment
     }
 
+    // Global document references are not supported
```

Comment:
What are the implications of this?

Reply:
just that you can't resolve the ref with typegen, it will end up as `unknown`